Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
Traceback (most recent call last):
  File "/home/alarcos/EETHAN/SE8/k-nucleotide/knucleotide.python3-3.py", line 6, in <module>
    from os import cpu_count
TypeError: 'module' object is not subscriptable
make: *** [Makefile:2: run] Error 1
